test/Table_Tests/src/Spreadsheet_Spec.enso

```diff
@@ -0,0 +1,30 @@
+from Standard.Base import all
+import Standard.Test
+import Standard.Table
+
+spec_fmt header extension read_method =
+    file = Enso_Project.data / ('multisheet.' + extension)
+    Test.group header <|
```

Comment:
I would suggest adding tests for loading empty sheets, as the logic for that in the Reader is not that trivial, so it would be good for the tests to check it.